### PR TITLE
Various cleanup in js files and 'extra for experts' -> 'extra for experienced'

### PR DESCRIPTION
**Pull request by peterasujan**

### CODE REVIEW

File: admin/bjc-topic.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ bjc['file'] = "";
  * e.g.   h1: Some Text [maybe/a/link/too]
  */
 //TODO put in bjc namespace
```
Comment (cycomachead) [inline admin/bjc-topic.js]: kill comment since it's done.:)

File: admin/quiz/multiplechoice.js [hunk 2]
```diff
@@ -1,110 +1,109 @@
...
+    
+    // save this MC dom element
+    // this.multipleChoice = $($(".MultipleChoice")[this.num]);
+    /*
+      var myDom = $("." + this.myClass).last();
+      this.multipleChoice = myDom;
+    */
+    //boolean to prevent shuffling after each answer submit
+    this.previouslyRendered = false;
+
```
Comment (cycomachead) [inline admin/quiz/multiplechoice.js:40]: Much whitespace.

File: cur/programming/robots/finch-pong.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
    <head>
         <script type="text/javascript" src="/bjc-r/admin/bjc-loader.js"></script>
       
-        <title>Extra for Experts: Pong</title>
+        <title>Extra for Experienced: Pong</title>
```
Comment (cycomachead) [inline cur/programming/robots/finch-pong.html]: Shouldn't these be for "Experience" and not "Experienced".
Past tense implies they're only for good people already.

**Comment by cycomachead** [comment]
Much commits. Such changes. Amaze.

:p

Way to get all those lines changed by just shifting indentation! ;)

**Comment by cycomachead** [comment]
URL for smaller change set, hiding whitespace only changes:
https://github.com/beautyjoy/bjc-r/pull/219/files?w=0

I noticed, we have a lot of `==` checks. Should these be `===`?

**Comment by peterasujan** [comment]
Uh. Good question...

File: admin/BJC.css [hunk 4]
```diff
@@ -64,9 +64,9 @@ a:visited {color:#36BBCE;}
 
 #full-bottom-bar {
 	background-color:#ccc;
-	background-image: url("../img/gobo.gif");
+	background-image: url("../img/costumes/alonzo.gif");
 	background-repeat:no-repeat;
-	background-position:top left;
+	background-position: 1px 2px;
```
Comment (cycomachead) [inline admin/BJC.css:69]: Boo!

Well, it's not terrible, but px values seem harder to maintain.

**Pull request merged**